bindings/python/tests/test_ref_type.py

```diff
@@ -6,10 +6,12 @@
 thisdir = Path(__file__).resolve().parent
 indir = thisdir / "input"
 
-dlite.storage_path.append(indir / "test_ref_type.json")
+dlite.storage_path.append(indir)
+dlite.storage_path.append(indir/"test_ref_type_middle.yaml")
 
+# Note, we read Middle from yaml file, which contains v0.2
 Top = dlite.get_instance("http://onto-ns.com/meta/0.1/Top")
-Middle = dlite.get_instance("http://onto-ns.com/meta/0.1/Middle")
+Middle = dlite.get_instance("http://onto-ns.com/meta/0.2/Middle")
```

Comment:
I think we should still test json as well

Reply:
We are still testing JSON, since all the other datamodels are read from `input/test_ref_type.json`.